frontend/packages/knative-plugin/src/components/add/import-types.ts

```diff
@@ -1,12 +1,15 @@
-export enum EventSources {
+export enum KnownEventSources {
   CronJobSource = 'CronJobSource',
-  ContainerSource = 'ContainerSource',
   ApiServerSource = 'ApiServerSource',
   KafkaSource = 'KafkaSource',
-  CamelkSource = 'CamelSource',
   SinkBinding = 'SinkBinding',
 }
 
+export enum EditorType {
+  YAML = 'YAMLEditor',
+  Form = 'FormEditor',
+}
+
```

Comment:
if use `type` for conditional validation schema , we may not need this as well

Reply:
done